README.md

```diff
@@ -26,12 +26,18 @@ default['supermarket_omnibus']['package_repo'] = 'chef-current'
 default['supermarket_omnibus']['package_url'] = 'http://bit.ly/98K8eH'
 ```
 
-If you wish to use custom SSL certificates, define the following `config` attributes:
+If you wish to specify additional settings, you can pass them via the `default['supermarket_omnibus']['config']` attribute.  
+Example: for custom SSL certificates define the following `config` attributes:
 
 ```ruby
 default['supermarket_omnibus']['config']['ssl']['certificate'] = '/full/path/to/ssl.crt'
 default['supermarket_omnibus']['config']['ssl']['certificate_key'] = '/full/path/to/ssl.key'
 ```
+Above attributes, if defined in supermarket.rb directly, would look like this:
```

Comment:
The Supermarket omnibus package works differently than our others,  it expects an `/etc/supermarket/supermarket.json` file which overrides the things in `supermarket.rb`  - that's why this cookbook writes out the JSON file and leaves the other one alone.

I worry that this paragraph will confuse users, because they won't see their expected attributes set in the right file.   I recommend changing this section to discuss the JSON file (and perhaps explain why)